electron-init.ts

```diff
@@ -16,108 +24,14 @@ function loadConfig(path) {
     return JSON.parse(data);
 }
 
-// Attempts to load the config file for the renter or provider.
-// configType should be 'renter' or 'provider'
-function tryLoadConfig(configType) {
-    console.assert(['renter', 'provider']
-        .some(e => e === configType), 'unknown config type');
-    if (skybinHome === null) {
-        return {
-            'error': 'SkyBin home is not known. Has SkyBin been setup yet?',
-        };
-    }
-    try {
-        return loadConfig(`${skybinHome}/${configType}/config.json`);
-    } catch (error) {
-        return {
-            'error': `Error loading config. Error: ${error}`,
-        };
-    }
-}
-
-// Register IPC handlers for main<->renderer process communication.
-ipcMain
-    .on('isSkybinSetup', (event) => {
-        event.returnValue = isSkybinSetup;
-    })
-    .on('setupSkybin', (event, setupOptions) => {
-        try {
-            event.returnValue = setupSkybin(setupOptions);
-        } catch (error) {
-            console.error('error setting up skybin', error);
-            event.returnValue = {
-                wasSuccessful: false,
-                errorMessage: `setup exception: ${error}`,
-                errorReason: 'unknown error',
-            };
-        }
-    })
-    .on('loadRenterConfig', (event) => {
-       event.returnValue = tryLoadConfig('renter');
-    })
-    .on('loadProviderConfig', (event) => {
-        event.returnValue = tryLoadConfig('provider');
+// Attempts to ping the renter daemon running at the
+// given address in 'host:port' form. Calls callback
+// with the resulting error (if one occurred).
+function pingRenter(address, callback) {
```

Comment:
Previously we discussed usage of a lock file to indicate the running status of the daemons. What caused the switch?